applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/GrantApplications/Index.js

```diff
@@ -776,11 +788,101 @@
             data: 'organizationName',
             className: 'data-table-header',
             render: function (data) {
-                return data ?? '';
+                return data || '{Organization Name}';
```

Comment:
why swap to falsey? i think for these fields and the fields below nullish (??) would make more sense.

Reply:
@AndreGAot  Updated;